probe/host/system_test.go

```diff
@@ -0,0 +1,38 @@
+package host_test
```

Comment:
These tests are a bit weak; why did you remove the original unit tests (that mock syscall.Uname and Readfile?)